pkg/config/containers.conf

```diff
@@ -164,6 +164,12 @@ default_sysctls = [
 #
 #ipcns = "shareable"
 
+# Default way to set an interface name inside container. Defaults to legacy
+# pattern of ethX, where X is a integer, when left undefined.
+# Options are:
+#  "device"   Uses the network_interface name from the network config as interface name. Falls back to the ethX pattern if the network_interface is not set.
```

Comment:
Can you add a line break so the line is not to long, we don't have strict a limit but I would prefer it to not be longer than the existing lines.

Reply:
Done